code/modules/mob/mob.dm

```diff
@@ -1528,12 +1534,15 @@ Use this proc preferably at the end of an equipment loadout
 			canmove = 0
 			lying = (category.flags & LOCKED_SHOULD_LIE) ? TRUE : FALSE //A lying value that !=1 will break this
 
-
-	else if(isUnconscious() || knockdown || paralysis || resting || !can_stand)
+	else if(resting || !can_stand)
```

Comment:
why do you put this in another `else` block if the executed code is exactly the same......

Reply:
isKnockedDown() will be overriden by a custom simple_animal check if there is a client present, meaning that if it wasn't for the separated check that does exactly the same resting would do nothing.

Reply:
```dm
else if(resting || !can_stand)
		stop_pulling()
		lying = 1
		canmove = 0
	else if(isKnockedDown())
		stop_pulling()
		lying = 1
		canmove = 0
```
In what universe is this different from:
```dm
else if(resting || !can_stand || isKnockedDown())
		stop_pulling()
		lying = 1
		canmove = 0
```

Reply:
Right